tests/integration/cairo1_compiler_bindings/print_test/simple/lib.cairo

```diff
@@ -0,0 +1,12 @@
+#[contract]
+mod SimpleContract {
+    use debug::PrintTrait;
+
+    #[external]
+    fn perform(a: felt252, b: felt252, c: felt252) {
+        123.print();
+        a.print();
+        b.print();
+        c.print();
+    }
+}
```

Comment:
Is it still used anywhere?

Reply:
[remove redundant code](https://github.com/software-mansion/protostar/pull/1803/commits/1ade6669e87f9c91a6b8148a0a5f83371168bd2f)